src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -359,13 +359,26 @@ public void sendClusterSettingsRequest(TransportService transportService) {
      *
      * @param transportService  The TransportService defining the connection to OpenSearch.
      */
-    public void sendLocalNodeRequest(TransportService transportService) {
-        sendGenericRequestWithExceptionHandling(
-            transportService,
-            ExtensionsOrchestrator.RequestType.REQUEST_EXTENSION_LOCAL_NODE,
-            ExtensionsOrchestrator.REQUEST_EXTENSION_LOCAL_NODE,
-            new LocalNodeResponseHandler()
-        );
+    public DiscoveryNode sendLocalNodeRequest(TransportService transportService) {
+        logger.info("Sending Local Node request to OpenSearch");
+        LocalNodeResponseHandler localNodeResponseHandler = new LocalNodeResponseHandler();
+        try {
+            transportService.sendRequest(
+                opensearchNode,
+                ExtensionsOrchestrator.REQUEST_EXTENSION_LOCAL_NODE,
+                new ExtensionRequest(ExtensionsOrchestrator.RequestType.REQUEST_EXTENSION_LOCAL_NODE),
+                localNodeResponseHandler
+            );
+            // Wait on local node response
+            localNodeResponseHandler.awaitResponse();
+        } catch (InterruptedException e) {
+            logger.info("Failed to recieve Local Node response from OpenSearch", e);
+        } catch (Exception e) {
+            logger.info("Failed to send Local Node request to OpenSearch", e);
```

Comment:
Should these failure log messages be higher than "info" level?

Reply:
Ill modify this so that the log messages are of level ERROR